frontend/components/campaigns/campaignCharities/CampaignCharityAccordionCard.tsx

```diff
@@ -0,0 +1,67 @@
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import { Box, Stack, Typography, useMediaQuery, useTheme } from '@mui/material';
+import Accordion from '@mui/material/Accordion';
+import AccordionDetails from '@mui/material/AccordionDetails';
+import AccordionSummary from '@mui/material/AccordionSummary';
+import {
+  accordianSx,
+  charityDesktopDescriptionSx,
+  charityDesktopImageSx,
+  charityGraphStackSx,
+  charityLogoSx,
+  charityMobileDescriptionSx,
+  charityMobileImageSx,
+  graphSx,
+  stackSx,
+} from '../../../styles/components/charities/CampaignCharityAccordianCard';
+import { CampaignCharityDonationPublicData } from '../../../types/campaignCharities';
+import SmallCompetingGraph from '../../charts/SmallCompetingGraph';
+
+interface Props {
+  campaignCharity: CampaignCharityDonationPublicData;
+}
+
+const CampaignCharityAccordionCard = ({ campaignCharity }: Props) => {
```

Comment:
Can you add a heading here that says "About the charity". It's confusing without context and there's also a lack of sectioning to guide the user. Also where's the link to go to the charity website?

<img width="388" alt="Screenshot 2022-11-02 at 2 12 06 PM" src="https://user-images.githubusercontent.com/71375383/199411978-3b57adcc-2871-4663-b353-7a7b93db1840.png">